django/db/models/sql/compiler.py

```diff
@@ -957,7 +1007,7 @@ def execute_sql(self, result_type):
             if is_empty:
                 rows = aux_rows
                 is_empty = False
-        return rows
+        return (rows, returning_values)
```

Comment:
OK. I already have them returning at a lower level but public update method still just returns the number of rows affected. I'll look into the `bulk_create()` case.

Thanks